target/linux/ramips/dts/mt7621_dlink_dir-1960-a1.dts

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_dlink_dir-xx60-a1.dtsi"
+
+/ {
+	compatible = "dlink,dir-1960-a1", "mediatek,mt7621-soc";
+	model = "D-Link DIR-1960 A1";
+
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_white;
+		led-running = &led_power_white;
+		led-upgrade = &led_net_orange;

```

Comment:
I don't think using the net LED is wise here, as a blinking net LED can be interpreted differently.

Reply:
This actually follows the behavior of the stock firmware, which also blinks the orange net LED while updating the firmware.

Reply:
Will this actually work, i.e. will the DT setup really overwrite the user-space setup as LAN led when upgrading?

Despite, this would only make sense if you limit the net LED to the link state and remove blinking on rx/tx.

Reply:
It works fine because the orange net LED is unused on OpenWrt apart from indicating firmware upgrade. Link on, RX and TX activity are mapped to the green net LED (which apart from being white on the DIR-1760/1960/2660 is the same GPIO).

Reply:
Sorry, you are right and I was talking nonsense (partly mixed this up with another PR).

Might only be a problem when a user sets up something to that LED manually.

I'd personally still not use a LED designated for ethernet here when there are actually two "power" LEDs available. But that's the author's design choice now.

Reply:
I'd personally leave as it is. It matches the stock firmware behavior (both for booting and also for upgrading), works out of the box with the default OpenWrt settings and it's exactly how it's currently configured on the DIR-867/878/882 DTS. And parting from the premise of "the user might manually setup the LED to do something different", the same is valid for any of the available LEDs, including the power LEDs and the WiFi LEDs...